mini-wallet.md

```diff
@@ -1,33 +1,26 @@
-## Try MiniWallet
+# Diem MiniWallet
 
-Install the Diem Python SDK with MiniWallet and MiniWallet Test Suite (included since version 1.2.6):
-```
-pip install diem[all]
-```
-You may need to use quotes `pip install 'diem[all]'`.
+## Introduction
 
-To include the Diem MiniWallet in a pip requirements.txt file add:
-```
-diem[all]>=1.2.6
-```
+The Diem MiniWallet is a simplified wallet that can be run from the command line. You can use the Diem MiniWallet (and the included testing suite) to test your wallet application and to help develop your app to meet our requirements. 
 
+The Diem MiniWallet connects to the Diem testnet by default. If you want to use the Diem MiniWallet to test in your local test network, read the instructions here. 
 
-Once the pip install command above has run, the `dmw` cli will be installed in your pip enviroment.
+### Install the Diem MiniWallet
 
-To see available sub-commands:
-```
-dmw --help
-```
+1. Install the Diem Python SDK with the MiniWallet and it’s Test Suite (both included since version 1.2.6) using this command: `pip install diem[all]`. You may need to use quotes around `'diem[all]'`. 
+
+To include the Diem MiniWallet in a pip requirements.txt file add:
+`diem[all]>=1.2.6`
```

Comment:
may move this to the end of `1.` paragraph, otherwise the rendered result looks odd, as above line has no padding at left.